Automattic/wp-calypso
# Signup reskin: UI changes for the user step

#### Changes proposed in this Pull Request

This PR is towards implementing the reskinned signup experiment. For more context on the experiment, read pbxNRc-IO-p2.

NOTE:

This PR implements reskin UI changes for the user step. Being the final PR of the batch, this PR also represents the end to end reskinned signup flow experience. 

Here's the [Figma file](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) for reference.

For the record:
Parent PR: https://github.com/Automattic/wp-calypso/pull/51611 and root PR: https://github.com/Automattic/wp-calypso/pull/50512

### Desktop

<img width="1054" alt="Screenshot 2021-04-02 at 8 30 12 PM" src="https://user-images.githubusercontent.com/1269602/113434622-ee110080-93fe-11eb-80a3-92f7b47cd42c.png">

### Tablet

![tab](https://user-images.githubusercontent.com/1269602/113434799-3c260400-93ff-11eb-9c37-0aa0da7d7ad9.jpg)


### Mobile

![calypso localhost_3000_start_user(Pixel 2)](https://user-images.githubusercontent.com/1269602/113434660-fff2a380-93fe-11eb-8fec-fe249f64645d.png)


#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start.
* Verify that the user step UI matches with the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) in desktop, tablet, and mobile viewports.
* Verify that the log in link in user registration step redirects to the login page.
* Verify that all the signup steps - including the domains and plans steps - match the Figma design for reskinned UI, in desktop, tablet, and mobile views.
* Assign yourself to control in Abacus, and verify that the control group experience is unaffected.
